<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>721</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Cohort Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBoxAtlas">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Brain Atlas</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="rightMargin">
              <number>100</number>
             </property>
             <item>
              <widget class="QLabel" name="labelAtlasName">
               <property name="text">
                <string>Brain Atlas name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelRegionNumber">
               <property name="text">
                <string>Brain region number = </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="btnSelectAtlas">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Select Atlas</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnViewAtlas">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>View Atlas</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="textCohortName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>MRI Cohort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="GroupTableWidget" name="groupTableWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>850</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="tabGroup">
         <attribute name="title">
          <string>Groups and Demographics</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="GroupsAndDemographicsWidget" name="groupsAndDemographicsWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabSubject">
         <attribute name="title">
          <string>Subject Data</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="SubjectDataWidget" name="subjectDataWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabAverages">
         <attribute name="title">
          <string>Group Averages</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="GroupAveragesWidget" name="groupAveragesWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabBrain">
         <attribute name="title">
          <string>Brain View</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="BrainAtlasWidget" name="brainWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuGroups">
    <property name="title">
     <string>Groups</string>
    </property>
    <addaction name="actionLoad_subject_group_from_file"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_group"/>
    <addaction name="actionRemove_group"/>
    <addaction name="actionMove_group_up"/>
    <addaction name="actionMove_group_down"/>
    <addaction name="separator"/>
    <addaction name="actionInvert"/>
    <addaction name="actionMerge"/>
    <addaction name="actionIntersect"/>
   </widget>
   <widget class="QMenu" name="menuSubjects">
    <property name="title">
     <string>Subjects</string>
    </property>
    <addaction name="actionSelect_all_subjects"/>
    <addaction name="actionClear_selection"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_subject"/>
    <addaction name="actionAdd_subject_above"/>
    <addaction name="actionAdd_subject_below"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_subject"/>
    <addaction name="separator"/>
    <addaction name="actionMove_subject_up"/>
    <addaction name="actionMove_subject_down"/>
    <addaction name="actionMove_subject_to_top"/>
    <addaction name="actionMove_subject_to_bottom"/>
    <addaction name="separator"/>
    <addaction name="actionNew_group_from_selection"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionGroups_and_Demographics"/>
    <addaction name="actionSubject_Data"/>
    <addaction name="actionGroup_Averages"/>
    <addaction name="actionBrain_View"/>
   </widget>
   <widget class="QMenu" name="menuBrain_View">
    <property name="title">
     <string>Brain View </string>
    </property>
    <addaction name="actionGenerate_figure"/>
   </widget>
   <widget class="QMenu" name="menuGraph_Analysis">
    <property name="title">
     <string>Graph Analysis</string>
    </property>
    <addaction name="actionNew_MRI_graph_analysis"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGroups"/>
   <addaction name="menuSubjects"/>
   <addaction name="menuView"/>
   <addaction name="menuBrain_View"/>
   <addaction name="menuGraph_Analysis"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/open.png</normaloff>:/buttons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/save.png</normaloff>:/buttons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionLoad_subject_group_from_file">
   <property name="text">
    <string>Load subject group from file ...</string>
   </property>
  </action>
  <action name="actionMove_group_up">
   <property name="text">
    <string>Move up</string>
   </property>
  </action>
  <action name="actionMove_group_down">
   <property name="text">
    <string>Move down</string>
   </property>
  </action>
  <action name="actionAdd_group">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove_group">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionSelect_all_subjects">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionAdd_subject">
   <property name="text">
    <string>Add subject</string>
   </property>
  </action>
  <action name="actionAdd_subject_above">
   <property name="text">
    <string>Add above</string>
   </property>
  </action>
  <action name="actionAdd_subject_below">
   <property name="text">
    <string>Add below</string>
   </property>
  </action>
  <action name="actionRemove_subject">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionMove_subject_up">
   <property name="text">
    <string>Move up</string>
   </property>
  </action>
  <action name="actionMove_subject_down">
   <property name="text">
    <string>Move down</string>
   </property>
  </action>
  <action name="actionMove_subject_to_top">
   <property name="text">
    <string>Move to top</string>
   </property>
  </action>
  <action name="actionMove_subject_to_bottom">
   <property name="text">
    <string>Move to bottom</string>
   </property>
  </action>
  <action name="actionGroups_and_Demographics">
   <property name="text">
    <string>Groups and Demographics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionSubject_Data">
   <property name="text">
    <string>Subject Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionGroup_Averages">
   <property name="text">
    <string>Group Averages</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionBrain_View">
   <property name="text">
    <string>Brain View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionGenerate_figure">
   <property name="text">
    <string>Generate figure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNew_MRI_graph_analysis">
   <property name="text">
    <string>New MRI graph analysis ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionClear_selection">
   <property name="text">
    <string>Clear selection</string>
   </property>
  </action>
  <action name="actionNew_group_from_selection">
   <property name="text">
    <string>New group from selection</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
  <action name="actionMerge">
   <property name="text">
    <string>Merge</string>
   </property>
  </action>
  <action name="actionIntersect">
   <property name="text">
    <string>Intersect</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrainAtlasWidget</class>
   <extends>QGraphicsView</extends>
   <header>braphy.gui.widgets.brain_atlas_widget</header>
  </customwidget>
  <customwidget>
   <class>GroupTableWidget</class>
   <extends>QWidget</extends>
   <header>braphy.gui.widgets.group_table_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GroupsAndDemographicsWidget</class>
   <extends>QWidget</extends>
   <header>braphy.gui.widgets.groups_and_demographics_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubjectDataWidget</class>
   <extends>QWidget</extends>
   <header>braphy.gui.widgets.subject_data_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GroupAveragesWidget</class>
   <extends>QWidget</extends>
   <header>braphy.gui.widgets.group_averages_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
